apps/desktop/src/renderer/globals.css

```
@@ -522,4 +522,39 @@
+		}
+	}
+	.animate-shine {
+		--shine-peak: rgba(255, 255, 255, 0.92);
+		background-image: linear-gradient(
+			110deg,
+			currentColor 0%,
+			currentColor 35%,
+			var(--shine-peak) 50%,
+			currentColor 65%,
+			currentColor 100%
+		);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)